person.rb

```diff
@@ -32,5 +32,5 @@ def can_use_services?
   end
 end
 
-me = Person.new(14, 'donard')
+me = Person.new(14, 'Isabel')
 p me.validate_name
```

Comment:
when I run your code I get this error : 
![image](https://user-images.githubusercontent.com/43698511/144401831-e0401a42-c34b-4669-977d-9c6d096ad013.png)
I think the problem is in line 25 `@name = @corrector.correct_name` the method correct_name is expecting an argument as defined in the corrector class.

- [ ] Kindly fix the issue by passing the @name as argument.

Reply:
This issue has been addressed well. Great job! :clap: :+1: